api/proto/teleport/legacy/types/types.proto

```diff
@@ -2428,6 +2442,13 @@ message AccessRequestFilter {
   string User = 2 [(gogoproto.jsontag) = "user,omitempty"];
   // RequestState filters for requests in a specific state.
   RequestState State = 3 [(gogoproto.jsontag) = "state,omitempty"];
+  // SearchKeywords is a list of search keywords to match against resource field values.
+  repeated string SearchKeywords = 4 [(gogoproto.jsontag) = "search,omitempty"];
+  // Scope is an aditional filter to view requests based on needs review, reviewed, my requests
+  AccessRequestScope Scope = 5 [(gogoproto.jsontag) = "scope,omitempty"];
```

Comment:
This scope filter overlaps with the User field, right?

Eg, what if I filter for UserX but then set the scope to `MY_REQUESTS`
Is it going to return 0 results?